lib/cc/analyzer/formatters/formatter.rb

```diff
@@ -7,7 +7,7 @@ def initialize(filesystem, output = $stdout)
           @output = output
         end
 
-        def write(data)
+        def write(_)
```

Comment:
Nice, can you `_data` so we don't lose the information about what it would be? Same with `_output`.